qa/suites/upgrade/quincy-x/filestore-remove-check/0-cluster/start.yaml

```diff
@@ -1,7 +1,7 @@
 meta:
 - desc: |
    Run ceph on one nodes,
-   Use xfs beneath the osds. upgrade to reef
+   Use xfs beneath the osds. upgrade to quincy
```

Comment:
Why?  It _is_ upgrading to reef which is where filestore was dropped.

Reply:
that was a typo, sorry, fixed